test/renderReact-test.js

```diff
@@ -30,9 +32,44 @@ describe('renderReact', () => {
       global.window = window;
       global.document = window.document;
 
+      const hydrateMethod = sinon.spy(ReactDOM, 'hydrate');
```

Comment:
will this work when `hydrate` doesn't exist yet?

Reply:
@ljharb yes, this test runs in the hypernova-react package which (in this PR) depends on React ^16 and therefore always has the hydrate method available.
https://github.com/airbnb/hypernova-react/pull/30/files

Reply:
hypernova-react doesn't *depend* on v16, it devDepends on it, and peerDepends on 0.14 or 0.15 or 16 - if we wanted to test this repo on 0.14 or 0.15 (which we should, if we're not already), this test would fail.

Reply:
how will we switch the react version to in our test cases? Shouldn't we use v16 and mock v15/v0.14 behaviour, which I'm doing on [line 59](https://github.com/airbnb/hypernova-react/pull/30/files#diff-1b15a2ce9bd6b5e1cd01f181d7ca34a5R59)

Reply:
I think this is fine for this PR :-) when we actually add tests on all the React versions, we wouldn't want to mock things out - mocks make tests more brittle.